app/views/users/index.html.erb

```diff
@@ -13,11 +13,9 @@
 
 <% @users.each do |user| %>
   <tr>
-    <td><%= user.name %></td>
+    <td><%= link_to user.name, user_path(user) %></td>
```

Comment:
I also don't think it's super-important to move the "Show" link to be on the user's name directly; it's a minor improvement.